production/helm/loki/templates/loki-canary/daemonset.yaml

```diff
@@ -2,7 +2,7 @@
 {{- if .enabled -}}
 ---
 apiVersion: apps/v1
-kind: DaemonSet
+kind: {{ .kind }}
```

Comment:
As I know, `updateStrategy` is not supported field on Deployment, instead `strategy` must be used. And I miss the replica setting, if kind is Deployment.

Reply:
added the strategy and replicas